include/drivers/sensor.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_SENSOR_H
+#define DRIVERS_SENSOR_H
+
+#include <dm.h>
+#include <intrusive.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <util.h>
+
+#define SENSOR_OPS(dev) \
+	(&container_of((dev)->drv, struct sensor_driver, drv)->ops)
+
+struct sensor_info {
+	const char *const name;       /**< Name exported to SCPI. */
+	const uint8_t     offset;     /**< offset of register. */
+	const int16_t     min_value;  /**< Addition factor. */
+	const int16_t     multiplier; /**< Multiplicaton factor. */
+};
+
+struct sensor_driver_ops {
+	struct sensor_info *(*get_info)(struct device *dev, uint8_t id);
+	int                 (*read_raw)(struct device *dev, uint8_t id,
+	                                uint32_t *raw);
+};
+
+struct sensor_driver {
+	const struct driver            drv;
+	const struct sensor_driver_ops ops;
+};
+
+static inline struct sensor_info *
+sensor_get_info(struct device *dev, uint8_t id)
```

Comment:
Where did the comment for this function go? You seem to have lost it with the other one

Reply:
Fixed